suuchi-core/src/main/proto/suuchi.proto

```diff
@@ -57,4 +57,19 @@ message Shard {
 message Node {
     string host = 1;
     int32 port = 2;
+}
+
+service SuuchiScan {
+    rpc Scan (ScanRequest) returns (stream ScanResponse);
+}
+message ScanRequest {
+    int32 start = 1;
```

Comment:
It would really help to extract the `start` and `end` as `TokenRange`. Given we had plans to use the same thing with other store methods too.